pkg/errors/errors.go

```diff
@@ -364,18 +484,29 @@ func IsIPOutOfRange(err error) bool {
 	return checkError(err, IPOutOfRange)
 }
 
+func IsMultipleErrors(err error) bool {
```

Comment:
Could you please point me to an example usage of this function as well?

Reply:
IsMultipleErrors doesn't currently have any usage. Mainly added since all errors have this functionality in the errors pkg.

Reply:
Closing to help me organize PR revisions, feel free to re-open if I didn't address something.